fab_test.py

```diff
@@ -20,9 +22,15 @@
 
 
 # paths
+
 go_path = "$HOME/go"
+
+if platform == "Darwin":
+    go_root = "/usr/local/Cellar/go/1.10.2/libexec"
+else:
+    go_root = "/usr/local/go"
+
```

Comment:
@mhchia should we define go_root using:

```bash
$ go env GOROOT
/usr/local/go
```
???

Reply:
Sure, it will be cleaner.

Reply:
Tracked in https://github.com/mhchia/test-fabric/issues/4